external-import/misp/src/misp.py

```diff
@@ -59,8 +59,6 @@
 class CryptocurrencyWallet:
     """Cryptocurrency wallet observable."""
 
-    pass
```

Comment:
Please leave the `pass` statements, since I think it is a good approach for indicating an empty class. Even if the present comments make the code executable. Further explanation: https://stackoverflow.com/a/49754261